tests/platform_helper/utils/fixtures/addons_files/alb_addons.yml

```diff
@@ -0,0 +1,7 @@
+my-alb:
+  type: alb
+  environments:
+    dev:
+      cdn_domains_list:
+        dev.app.uktrade.digital: ["internal", "app.uktrade.digital" ]
+      default_waf: "waf_sentinel_012345678910_default"
```

Comment:
I think we need "all the things" in here so that we can be sure they are all allowed.

Reply:
done